### ISSUE TITLE
Format Exception

### ISSUE THREAD

**Issue by ObaidGH** [AlexxEG/YouTube-Downloader#36]
`System.FormatException` encountered at line: 106 of PlaylistReader.cs

``` c#
if ((m = _regexPlaylistIndex.Match(line)).Success)
{
     _currentVideoPlaylistIndex = int.Parse(m.Groups[1].Value);
}
```

**Comment by AlexxEG** [comment]
Could you try again with 58e507b55a650c6b6a0c6450bc45503a36c5c1d3 (EDIT: and 779f8a8f4e2e40a156496302cfe4beab1353348c) so that I can get a more detailed exception?

I'm going to need the exception message.

**Comment by ObaidGH** [comment]
Still the problem persists

![exc](https://cloud.githubusercontent.com/assets/14979775/19835571/12648850-9eb1-11e6-8f36-dbe6a158c590.PNG)

**Comment by AlexxEG** [comment]
I think I've fixed it in 9016e6a57e4003870838f2272cbe4b6a6004e8c9. Try again :)
